nvflare/app_opt/statistics/visualization/statistics_visualization.py

```diff
@@ -57,25 +57,31 @@ def _prepare_histogram_data(self, data, display_format="percent", white_list_fea
 
         for feature in target_features:
             xs = data[feature]["histogram"]
-            counts = data[feature]["count"]
             hists = {}
             feature_edges[feature] = []
             for i, ds in enumerate(xs):
                 ds_hist = xs[ds]
                 ds_bucket_counts = []
+
                 for bucket in ds_hist:
                     if i == 0:
                         feature_edges[feature].append(bucket[0])
                     if display_format == "percent":
-                        ds_bucket_counts.append(round(bucket[2] / counts[ds], 2))
+                        sum_value = self.sum_counts_in_histogram(ds_hist)
+                        ds_bucket_counts.append(round(bucket[2] / sum_value, 2))
                     else:
                         ds_bucket_counts.append(bucket[2])
```

Comment:
For bucket[0] and bucket[2] where does the index come from?
What is the structure for this ds_hist?